File: LEETCODE/trees/traversalTechniques/DeptFirstSearch.java/PostorderTraversal.java
import LEETCODE.trees.BinaryTree;

public class PostorderTraversal {
    private static void postorder(BinaryTree.Node root){
        if(root == null){
            return;
        }
        postorder(root.left);
        postorder(root.right);
        System.out.println(root.data);
    }

    public static void main(String[] args) {
        BinaryTree.Node  root = BinaryTree.createTree();
        postorder(root);
    }
}

File: LEETCODE/trees/traversalTechniques/DeptFirstSearch.java/PostorderTraversal.cpp
#include<bits/stdc++.h>
using namespace std;

struct Node{
    int data;
    Node* left;
    Node* right;
    Node(int data){
        this->data = data;
    }
};

int postorder(Node* root, vector<int> &ans){
    if(root == nullptr){
        return ;
    }
    postorder(root->left, ans);
    postorder(root->right, ans);
    ans.push_back(root->data);
}
int main(){
    
    return 0;
}